build_scripts/default/nginx_conf/sites/openrecords_v2_0.conf

```diff
@@ -42,9 +61,45 @@ server {
   }
 
   location / {
-    proxy_pass                          http://openrecords_v2_0;
+    if ($maintenance_mode = 0) {
+        set $openrecords_authentication "off";
+    }
+    auth_basic $openrecords_authentication;
+    auth_basic_user_file /vagrant/build_scripts/web_setup/nginx_conf/sites/.htpasswd;
+    if ($maintenance_mode = 1) {
+       add_header Set-Cookie "authorized_maintainer=true;max-age=3153600000;path=/";
+    }
+
     proxy_set_header                    Host $host;
     proxy_set_header                    X-Real-IP $remote_addr;
+    proxy_pass                          http://openrecords_v2_0;
+  }
+
+  error_page 401 @unauthorized;
+  location @unauthorized {
+    root /vagrant/build_scripts/web_setup/nginx_conf/sites;
```

Comment:
Change to default

Reply:
In this file we have the certificates stored in `/vagrant/build_scripts/web_setup/`. Should we keep it consistent? I can move the htpasswd file there as well.

As far as I can tell, that has been that way since we created the build scripts. (https://github.com/CityOfNewYork/NYCOpenRecords/commit/c186836ecc6eeef1a665284c283a22e9bc62f2bc)

Reply:
Let's stay consistent then and leave it as `/vagrant/build_scripts/web_setup/`